keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/KeyboardVisibilityEvent.java

```diff
@@ -49,6 +50,12 @@ public static Unregistrar registerEventListener(final Activity activity,
             throw new NullPointerException("Parameter:activity must not be null");
         }
 
+        int softInputMethod = activity.getWindow().getAttributes().softInputMode;
+        if(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE != softInputMethod &&
```

Comment:
seems that sofInputMode state is managed like [this](https://github.com/android/platform_frameworks_base/blob/master/core/java/android/view/WindowManager.java#L1400), so following code would be better

`(activity.getWindow().getAttributes().softInputMode & WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE) != 0`